docs/_posts/2018-10-11-524-release.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blogpost
+title:  "5.2.4 Release Announcement"
+section_id: blog
+date:   2018-10-11 12:00:00
+author: Open GEE Development Team
+---
+
+Hi Open GEE Community!
```

Comment:
Can you add an extra return or spaces here so the line isn't right up against the date icon